addons/medical_feedback/RscInGameUI.hpp

```diff
@@ -0,0 +1,14 @@
+
+class RscPictureKeepAspect;
+class RscInGameUI {
+    class RscStanceInfo {
+        controls[] += {QGVAR(BloodVolumeInfoIndicator)};
+        class GVAR(BloodVolumeInfoIndicator): RscPictureKeepAspect {
```

Comment:
Maybe better name is just `GVAR(bloodVolumeIndicator)`.

Reply:
I like it, but i thought it may be used in future for some other indicators, so added some extra word.

Reply:
Changed